scripts/retrieve_cutouts.py

```diff
@@ -0,0 +1,97 @@
+# SPDX-FileCopyrightText:  Open Energy Transition gGmbH
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import os
+import sys
+sys.path.append(os.path.abspath(os.path.join(__file__ ,"../../")))
+from pathlib import Path
+from google_drive_downloader import GoogleDriveDownloader as gdd
+import re
+from zipfile import ZipFile
+import yaml
+import logging
+import warnings
+warnings.filterwarnings("ignore")
+from scripts._helper import mock_snakemake, update_config_from_wildcards, create_logger, PYPSA_EARTH_DIR
+
+
+logger = create_logger(__name__)
+
+
+def download_and_unzip_gdrive(config, disable_progress=False):
+    """
+        Downloads and unzips USA cutouts to submodules/pypsa-earth/cutouts folder 
+    """
+    resource = config["category"]
+    file_path = os.path.join(PYPSA_EARTH_DIR, "tempfile.zip")
+    destination = os.path.join(PYPSA_EARTH_DIR, config["destination"])
+    url = config["urls"]["gdrive"]
```

Comment:
Hi, @GbotemiB, thank you for the comment. What type of argument? Initially I thought of giving directory to store, but it will be handled by bungle config. Also since this script will be used only for cutouts retrieval, I thought that there is no need for generalized function. I also thought of placing this function into `_helper.py` and use it at different data retrieval scripts. This can be done in future, as more scripts will be present.

Reply:
Ok, I see what you mean. URLs are set in `config.main.yaml` in custom_databundle section, and destination too. So we can set them in config, and refer as config["custom_databundles"]